ccd-definition/CaseEvent/Camunda/DocmosisEvents-nonprod.json

```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "GENERATE_RESPONSE_SEALED",
+    "Name": "Generate response sealed form",
+    "Description": "Generates sealed form for claim response",
+    "PreConditionState(s)": "*",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/about-to-submit",
```

Comment:
this should be - 
"CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/version/V_1/about-to-submit",

V1 is missing here